crates/bevy_pbr/src/render/mesh_bindings.rs

```diff
@@ -34,9 +37,12 @@ mod layout_entry {
             },
         }
     }
-    pub(super) fn model(binding: u32) -> BindGroupLayoutEntry {
-        let size = MeshUniform::min_size().get();
-        buffer(binding, size, ShaderStages::VERTEX | ShaderStages::FRAGMENT)
+    pub(super) fn model(render_device: &RenderDevice, binding: u32) -> BindGroupLayoutEntry {
+        GpuArrayBuffer::<MeshUniform>::binding_layout(
+            binding,
+            ShaderStages::VERTEX_FRAGMENT,
+            render_device,
+        )
```

Comment:
I guess `skinning` and `weights` can get the same treatment in a future PR right? Since they could use a storage buffer in lieu of a uniform buffer when available.

Reply:
Let me reformulate: we already use an index and an `Uniform` for skins and morphs, but is there any benefit to use a storage buffer when available?

Reply:
We can pack all the skinning data into 1 storage buffer, instead of multiple uniform buffers. Less buffers = less rebinds = more performance :)

Reply:
Right, it applies to all buffers of struct data, basically. One needs to plumb in the indices, so perhaps a per-object index has an index or offset into a skinning array or so. I don't know too much about those skins work to know how best to structure the data. Another aspect of having a single buffer is if the data itself repeats with varying counts of things (so things like vertices in vertex buffers, or maybe the number of transforms in a skin?) then some kind of allocator is usually used to manage loading in/out data without having to reallocate buffers all the time and rewrite everything, and handle things of varying sizes.